src/core/Object3D.js

```diff
@@ -670,6 +671,31 @@ Object.assign( Object3D.prototype, EventDispatcher.prototype, {
 
 			object.geometry = serialize( meta.geometries, this.geometry );
 
+			var parameters = this.geometry.parameters;
+
+			if ( parameters !== undefined && parameters.shapes !== undefined ) {
+
+				var shapes = parameters.shapes;
+				var uuids = [];
```

Comment:
What's this array for? 🤔

Reply:
Oh, this is a remnant of a previous solution. I'll remove it...